packages/utilities/src/overflowHandler/README.md

```diff
@@ -0,0 +1,51 @@
+# overflowHandler
+
+`overflowHandler` is a framework-agnostic utility for handling the truncation of
+a list of items.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```js
+import { createOverflowHandler } from '@carbon/utilities';
+
+createOverflowHandler({
+  container, // The document node that holds the items. HTMLElement
+  maxVisibleItems, // Maximum number of visible items. If provided, only this number of items will be shown. number | undefined
+  onChange: (visibleItems: HTMLElement[], hiddenItems: HTMLElement[]) => void, // Callback function invoked when the visible and hidden items change.
+  dimension, // The dimension to consider for overflow calculations. "width" | "height" | undefined
+});
+```
+
+The only required argument necessary to consume this utility is `container`.
+Once the handler has been instantiated, an event handler is created that checks
+the width of the container when it's rendered or resized. The handler adds the
+necessary `data-hidden` attributes to the items that should be hidden. These
+items can then be hidden using the following style in the implementation.
+
+```css
+[data-hidden]:not([data-fixed]) { // scope if needed
```

Comment:
Is `// scope if needed` supposed to be a code comment? If so, I suggest using `/* */` which is the official syntax for a comment in CSS. Alternatively, maybe it would be fine to just delete it?